docs/TESTS.md

```diff
@@ -42,6 +42,42 @@ Gradle project:    [C:\Users\johndoe\exercism\java\etl\build.gradle   ](...)
 
 ... and away you go!!!
 
+## Windows with Netbeans
+
+This could also apply, with slight variations, to all OSs that support the Netbeans IDE (Netbeans is a Java-based and is available for Windows, Mac OS X, and Linux). When downloading Netbeans, be sure to get the Java SE, Java EE, or "All" version. You may also need a JDK.
```

Comment:
No need to put the `(Netbeans is a Java-based` in parens here.  A separate sentence would be appropriate and preferable.  Also, there appears to be a typo: `Netbeans is a Java-based` should probably be `Netbeans is Java-based`.